src/app/views/login/SignInController.jsx

```diff
@@ -142,10 +142,12 @@ export class LoginController extends Component {
                 errorContents.passwordMessage = "Enter a password";
                 break;
             case "NotAuthorised":
-                errorContents.errorsForBody.push(<p>Email address or password are incorrect</p>);
+                errorContents.errorsForBody.push(<p key={"error-not-authorised"}>Email address or password are incorrect</p>);
                 break;
             case "TooManyFailedAttempts":
-                errorContents.errorsForBody.push(<p>You've tried to sign in to your account too many times. Please try again later.</p>);
+                errorContents.errorsForBody.push(
+                    <p key={"error-too-many-attempts"}>You've tried to sign in to your account too many times. Please try again later.</p>
```

Comment:
I think you don't meed `{` when you have sting :-) 
```suggestion
                    <p key="error-too-many-attempts">You've tried to sign in to your account too many times. Please try again later.</p>
```